main/static/css/cards.css

```diff
@@ -1,16 +1,21 @@
 .box{
   border: 3px solid #fff;
 }
+
```

Comment:
Why are these empty lines being added :confused:

Reply:
It made the code more readable for me. :P

Reply:
Please don't do these .